src/backend.github.js

```diff
@@ -253,13 +313,70 @@ var _ = Mavo.Backend.register($.Class({
 
 					if (this.repo) {
 						return this.request(`repos/${this.username}/${this.repo}`)
-							.then(repoInfo => {
-								if (this.branch === undefined) {
-									this.branch = repoInfo.default_branch;
+						.then(repoInfo => {
+							if (this.branch === undefined) {
+								this.branch = repoInfo.default_branch;
+							}
+
+							let params = (new URL(document.location)).searchParams;
+
+							if (!this.mavo.source && !params.get("source")) { // if url doesn't have source, check for forks
```

Comment:
`source` is not the only URL param, `source-APPID` also works. But why do you need to check the URL param? If it's valid, there will be a `this.mavo.source` which you're already checking.